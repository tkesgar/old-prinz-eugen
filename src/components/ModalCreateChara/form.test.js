import { mount } from '@vue/test-utils'
import localVue from '../../utils/test/local-vue'
import CreateCharaModalForm from './form'

describe('CreateCharaModalForm', () => {
  const wrapper = mount(CreateCharaModalForm, { localVue })

  test('should mount successfully', () => {
    expect(wrapper.isVueInstance()).toBe(true)
  })

  test('should match snapshot', () => {
    expect(wrapper.element).toMatchSnapshot()
  })
})
