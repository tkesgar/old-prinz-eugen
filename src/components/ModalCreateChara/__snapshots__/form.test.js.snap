// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CreateCharaModalForm should match snapshot 1`] = `
<form
  class=""
>
  <div
    class="form-group"
    id="__BVID__4"
    role="group"
  >
    <label
      class="d-block"
      for="CreateCharaModal_fullName"
      id="__BVID__4__BV_label_"
    >
      Nama karakter
    </label>
    <div>
      <input
        class="form-control"
        id="CreateCharaModal_fullName"
        type="text"
      />
      <!---->
      <!---->
      <!---->
    </div>
  </div>
   
  <div
    class="form-group"
    id="__BVID__6"
    role="group"
  >
    <label
      class="d-block"
      for="CreateCharaModal_nickName"
      id="__BVID__6__BV_label_"
    >
      Nama panggilan karakter
    </label>
    <div>
      <input
        class="form-control"
        id="CreateCharaModal_nickName"
        type="text"
      />
      <!---->
      <!---->
      <!---->
    </div>
  </div>
   
  <div
    class="form-group"
    id="__BVID__8"
    role="group"
  >
    <label
      class="d-block"
      for="CreateCharaModal_name"
      id="__BVID__8__BV_label_"
    >
      Nama halaman karakter
    </label>
    <div>
      <input
        aria-describedby="__BVID__8__BV_description_"
        aria-required="true"
        class="form-control"
        id="CreateCharaModal_name"
        maxlength="16"
        pattern="^\\\\w+$"
        required="required"
        type="text"
      />
      <!---->
      <!---->
      <small
        class="form-text text-muted"
        id="__BVID__8__BV_description_"
        tabindex="-1"
      >
        Nama halaman hanya boleh terdiri dari huruf, angka, dan garis bawah.
      </small>
    </div>
  </div>
  
  
  
  <button
    class="btn btn-primary btn-block"
    type="submit"
  >
    Buat karakter baru
  </button>
</form>
`;
